client/packages/common/src/intl/locales/en/common.json

```diff
@@ -770,6 +769,10 @@
   "label.doses-per": "Doses per",
   "label.doses-per-unit": "Doses per unit",
   "label.doses-per-vial": "Doses per vial",
+  "label.doses-plural_few": "doses",
+  "label.doses-plural_many": "doses",
+  "label.doses-plural_one": "dose",
+  "label.doses-plural_other": "doses",
```

Comment:
few and many shouldn't be needed...

Reply:
Not needed, but needed based on how we do the plurals in code >.>, else the translation won't show